src/api/admin/projects/projects-endpoint-admin.ts

```diff
@@ -24,9 +24,10 @@ export class ProjectsEndpointAdmin extends Endpoint {
     /**
      * Returns a list of all projects.
      */
-    getProjects(): Observable<ApiResponseData<ProjectsResponse> | ApiResponseError> {
-    
-        return this.httpGet("").pipe(
+    getProjects(zioRequest?: boolean): Observable<ApiResponseData<ProjectsResponse> | ApiResponseError> {
+        if (zioRequest === undefined) zioRequest = false;
```

Comment:
If it will not more harder to implement it also on the APP side I would op to make it required and with default value  as false, so if statement isn't necessary anymore:

```suggestion
    getProjects(zioRequest: boolean = false): Observable<ApiResponseData<ProjectsResponse> | ApiResponseError> {
```

You could also add the param to the docstring.

This suggestion IMO applies everywhere `zioRequest` us used.

Reply:
done in [f609cd7](https://github.com/dasch-swiss/dsp-js-lib/pull/485/commits/f609cd73b9e312532c23e6e6024178fac2e93850)